packages/migrations/src/index.ts

```diff
@@ -2,37 +2,26 @@
 import path from 'node:path';
 import url from 'node:url';
 import { createPool } from 'slonik';
-import { SlonikMigrator } from '@slonik/migrator';
 import { migrateClickHouse } from './clickhouse';
 import { createConnectionString } from './connection-string';
 import { env } from './environment';
+import { runPGMigrations } from './run-pg-migrations';
 
-const [, , cmd] = process.argv;
 const slonik = await createPool(createConnectionString(env.postgres));
+const [, , cmd] = process.argv;
 
 const __dirname = path.dirname(url.fileURLToPath(import.meta.url));
 const actionsDirectory = __dirname + path.sep + 'actions';
 console.log('Actions in:', actionsDirectory);
 
-const migrator = new SlonikMigrator({
-  migrationsPath: actionsDirectory,
-  slonik,
-  migrationTableName: 'migration',
-  logger: console,
-});
-
-// Why? We don't want to run the 'create' and 'down' commands programmatically, it should run from CLI.
-const isCreateCommand = cmd === 'create';
-const isDownCommand = cmd === 'down';
-
 // This is used by production build of this package.
 // We are building a "cli" out of the package, so we need a workaround to pass the command to run.
 
-if (env.isMigrator && !isCreateCommand && !isDownCommand) {
+if (env.isMigrator || cmd === 'up') {
```

Comment:
I think we can remove the if statement as we no longer run it as CLI

Reply:
done